compiler/damlc/tests/daml-test-files/InterfaceChoiceCollision.daml

```diff
@@ -2,41 +2,17 @@
 -- SPDX-License-Identifier: Apache-2.0
 
 -- @SINCE-LF-FEATURE DAML_INTERFACE
--- @ERROR name collision between choice InterfaceChoiceCollision:T.MyArchive (via interface InterfaceChoiceCollision:InterfaceA) and choice InterfaceChoiceCollision:T.MyArchive (via interface InterfaceChoiceCollision:InterfaceB)
+-- @ERROR name collision between choice InterfaceChoiceCollision:T.MyArchive (via interface InterfaceChoiceCollision1:Interface) and choice InterfaceChoiceCollision:T.MyArchive (via interface InterfaceChoiceCollision2:Interface)
 module InterfaceChoiceCollision where
```

Comment:
Can you link the issue #13915 in a comment, so we don't forget this is also a regression test.